docs/content/tutorial/customize-cluster.md

```diff
@@ -185,6 +185,105 @@ spec:
+
+#### Integration with User and Database Management 
+
+If you are creating users or databases, please see the [User/Database Management]({{< relref "tutorial/user-management.md" >}}) documentation. Databases created through the user management section of the spec can be referenced in your initialization sql. For example, if a database `zoo` is defined:
+
+```
+spec:
+  users:
+    - name: hippo
+      databases:
+       - "zoo"
+```
+
+You can connect to `zoo` by adding the following `psql` meta-command to your SQL:
+
+```
+\c zoo
+create table t_zoo as select s, md5(random()::text) from generate_Series(1,5) s;
+```
+
+#### Transaction support
+
+By default, `psql` commits each SQL statement as it completes. To combine multiple statements into a single [transaction](https://www.postgresql.org/docs/current/tutorial-transactions.html), use the [`BEGIN`](https://www.postgresql.org/docs/13/sql-begin.html) and [`COMMIT`](https://www.postgresql.org/docs/13/sql-commit.html) variables. 
+
+```
+BEGIN;
+create table t_random as select s, md5(random()::text) from generate_Series(1,5) s;
+COMMIT;
+```
+
+#### PSQL Exit Code and Database Init SQL Status
+
+The exit code from `psql` will determine when the `databaseInitSQL` status is set. When `psql` returns `0` the status will be set and SQL will not be run again. When `psql` returns with an error exit code the status will not be set. PGO will continue attempting to execute the SQL as part of its reconcile loop until `psql` returns normally. If `psql` exits with a failure, you will need to edit the file in your ConfigMap to ensure your SQL statements will lead to a successful `psql` return. The easiest way to make live changes to your ConfigMap is to use the following `kubectl edit` command:
+
+```
+kubectl -n <cluster-namespace> edit configmap hippo-init-sql
+```
+
+Be sure to transfer any changes back over to your local file. Another option is to make changes in your local file and use `kubectl --dry-run` to create a template and pipe the output into `kubectl apply`:
+
+```
+kubectl create configmap hippo-init-sql --from-file=init.sql=/path/to/init.sql --dry-run=client -o yaml | kubectl apply -f -
+```
+
+{{% notice tip %}}
+If you edit your ConfigMap and your changes aren't showing up, you may be waiting for PGO to reconcile your cluster. After some time, PGO will automatically reconcile the cluster or you can trigger reconciliation by applying any change to your cluster (e.g. with `kubectl apply -k kustomize/postgres`).
+{{% /notice %}}
+
+To ensure that `psql` returns a failure exit code when any of your statements fail, set the `ON_ERROR_STOP` variable, found [here](https://www.postgresql.org/docs/current/app-psql.html), as part of your SQL file:
+
+```
+\set ON_ERROR_STOP
+\echo `Any error will lead to exit code 3`
```

Comment:
nope that i not intentional

Reply:
Tested that out and am seeing the same behavior. I can remove the backticks...or i can remove the `echo` all together. I added it to provide a second example for meta-commands but it could be extra